tls-attestation/diagrams/kat-media-type-discovery.txt

```diff
@@ -0,0 +1,14 @@
+.-------.      .------------.          .-------.       .----------.
+| TLS A |      | Verifier X |          | TLS B |       | Attester |
+'---+---'      '-----+------'          '---+---'       '----+-----'
```

Comment:
Hmm, do we need TLS A and Verifier? I assume it's just for completeness

Reply:
I imagined we'd use them in sequence, with text blocks in between.  This is a way to provide the continuity across boundaries.